laplace/lllaplace.py

```diff
@@ -65,32 +65,32 @@ def __init__(self, model, likelihood, sigma_noise=1., prior_precision=1.,
                          backend_kwargs=backend_kwargs)
         self.model = FeatureExtractor(deepcopy(model), last_layer_name=last_layer_name)
         if self.model.last_layer is None:
-            self.mean = None
+            self.mean = prior_mean
             self.n_params = None
             self.n_layers = None
             # ignore checks of prior mean setter temporarily, check on .fit()
             self._prior_precision = prior_precision
             self._prior_mean = prior_mean
         else:
-            self.mean = parameters_to_vector(self.model.last_layer.parameters()).detach()
-            self.n_params = len(self.mean)
+            self.mean = prior_mean 
```

Comment:
I think this line should be `self.mean = self.prior_mean` after line 79 so the prior mean setter checks also apply.